astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,422 @@
+                ("center", "body"),
+                ("center_body", planet.capitalize()),
+                ("center_ansa", planet_defaults[planet]["center_ansa"]),
+                ("center_ew", "east"),
+                ("center_ra", ""),
+                ("center_ra_type", "hours"),
+                ("center_dec", ""),
+                ("center_star", ""),
+                ("viewpoint", viewpoint),
+                ("observatory", "Earth's center"),  # has no effect if viewpoint != observatory so can hardcode. no plans to implement calling observatories by name since ring node only names like 8 observatories
+                ("latitude", latitude),
+                ("longitude", longitude),
+                ("lon_dir", "east"),
+                ("altitude", altitude),
+                ("moons", planet_defaults[planet]["moons"]),
+                ("rings", planet_defaults[planet]["rings"]),
+                ("arcmodel", neptune_arcmodels[int(neptune_arcmodel)]),
+                ("extra_ra", ""),  # figure options below this line, can all be hardcoded and ignored
+                ("extra_ra_type", "hours"),
+                ("extra_dec", ""),
+                ("extra_name", ""),
+                ("title", ""),
+                ("labels", "Small (6 points)"),
+                ("moonpts", "0"),
+                ("blank", "No"),
+                ("opacity", "Transparent"),
+                ("peris", "None"),
+                ("peripts", "4"),
+                ("arcpts", "4"),
+                ("meridians", "Yes"),
+                ("output", "html"),
+            ]
+        )
+
+        # return request_payload if desired
+        if get_query_payload:
+            return request_payload
+
+        # query and parse
+        response = self._request(
+            "GET", self._url, params=request_payload, timeout=self.timeout, cache=cache
+        )
+
+        return response
+
+    def _parse_result(self, response, verbose=None):
+        """
+        Routine for parsing data from ring node
+
+        Parameters
+        ----------
+        self : RingNodeClass instance
+        response : list
+            raw response from server
+
+
+        Returns
+        -------
+        bodytable : `astropy.QTable`
+        ringtable : `astropy.QTable`
+        """
+        try:
+            self._last_query.remove_cache_file(self.cache_location)
+        except (FileNotFoundError, OSError):
+            # this is allowed: if `cache` was set to False, this
+            # won't be needed
+            pass
+
+        soup = BeautifulSoup(response.text, "html.parser")
+        text = soup.get_text()
+        textgroups = re.split("\n\n|\n \n", text)
+        ringtable = None
+        for group in textgroups:
+            group = group.strip(", \n")
+
+            # input parameters. only thing needed is epoch
+            if group.startswith("Observation"):
+                epoch = group.split("\n")[0].split("e: ")[-1].strip(", \n")
```

Comment:
without looking to much into the details, I think this line (and the previous ones) mess up the Windows tests, namely that your line-ending assumptions may not be true for windows.

Reply:
Thanks for looking into it @bsipocz, and yes, I see that the error comes from those lines.  I am pretty lost though as to how to play with or debug the code on a Windows machine.  In fact, it seems to only throw an error for specific Windows machines, since an undergrad I advise is working with the code on Windows and is not experiencing this issue.

Reply:
Have you tried using [`str.splitlines()`](https://docs.python.org/3/library/stdtypes.html#str.splitlines) instead of `str.split("\n")`?

Reply:
thanks for the suggestion!  I will look into it tomorrow.

Reply:
a version of this suggestion worked! thank you so much @eerovaher !  I *think* this is ready now

Reply:
I don't see the modified version with splitlines, could you please simplify the manual line ending checks from above and use the method as it was suggested?